### PR TITLE
Added script to build paping package on rhel 8.4

### PR DESCRIPTION
**Pull request by sachinkakatkar**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [x] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

File: p/paping/paping_rhel_8.4.sh [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# -----------------------------------------------------------------------------
+#
+# Package       : paping
+# Version       : v1.5.5
+# Source repo   : https://github.com/koolhazz/paping.git
+# Tested on     : RHEL 8.4
+# Script License: Apache License Version 2.0
+# Maintainer    : sachin.kakatkar@ibm.com
```
Comment (kandarpamalipeddi) [inline p/paping/paping_rhel_8.4.sh:8]: Please add `Language : ` field in the script header section.

You can find details in recent PRs.

**Review by kandarpamalipeddi — CHANGES_REQUESTED** [review]

File: p/paping/paping_rhel_8.4.sh (hunk 1 above)
Comment (sachinkakatkar) [reply p/paping/paping_rhel_8.4.sh:9]: Added Language section.

**Comment by sachinkakatkar** [comment]
Addressed all changes. @kandarpamalipeddi  Please review and close PR.

File: p/paping/paping_rhel_8.4.sh [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# -----------------------------------------------------------------------------
+#
+# Package       : paping
+# Version       : v1.5.5
+# Source repo   : https://github.com/koolhazz/paping.git
+# Tested on     : RHEL 8.4
+# Language      : C++
+# Script License: Apache License Version 2.0
+# Maintainer    : sachin.kakatkar@ibm.com
```
Comment (kandarpamalipeddi) [inline p/paping/paping_rhel_8.4.sh:9]: Updated the header as per the latest guidelines.
Missing "Travis-check" field in the header.

**Review by kandarpamalipeddi — CHANGES_REQUESTED** [review]

**Review by kandarpamalipeddi — APPROVED** [review]

**Pull request merged**